vault/core_test.go

```diff
@@ -1110,10 +1110,12 @@ func TestCore_LimitedUseToken(t *testing.T) {
 
 func TestCore_CleanLeaderPrefix(t *testing.T) {
 	// Create the first core and initialize it
-	inm := physical.NewInmemHA()
+	inm := physical.NewInmem()
+	inmha := physical.NewInmemHA()
```

Comment:
This is explicit for a reason -- to ensure that we are actually using separate backends in the tests (rather than one, and having it automatically be used for HA). This causes our unit tests to be utilized to test out this feature.